ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/HardFork/History/Qry.hs

```diff
@@ -528,6 +590,16 @@ epochToSize :: EpochNo -> Qry EpochSize
 epochToSize absEpoch =
   qryFromExpr (epochToSizeExpr absEpoch)
 
+-- | Translate 'PerasRoundNo' to the 'SlotNo' of the first slot in that Peras round
+--
+-- Additionally returns the length of the round.
+perasRoundNoToSlot :: PerasRoundNo -> Qry (SlotNo, PerasRoundLength)
+perasRoundNoToSlot perasRoundNo = qryFromExpr (perasRoundNoToSlotExpr perasRoundNo)
+
+-- | Translate 'SlotNo' to its corresponding 'PerasRoundNo'
+slotToPerasRoundNo :: SlotNo -> Qry PerasRoundNo
```

Comment:
It would also be useful (eg for https://github.com/tweag/cardano-peras/issues/101) to enrich this with the relative slot in the Peras round. See eg `slotToEpoch` for comparison.

Reply:
:+1: